unittests/test_policies.py

```diff
@@ -237,6 +237,9 @@ def test_runall(make_runner, make_cases, common_exec_ctx, tmp_path):
     with open(report_file, 'w') as fp:
         jsonext.dump(report, fp)
 
+    # Test handling of `None` in `time_total` in junit report
+    report['runs'][0]['testcases'][0]['time_total'] = None
```

Comment:
Yes, but here you are not testing the normal path anymore, are you?

Reply:
I am because there are more than one test cases running.

Reply:
I would take the last test case and expand a bit the comment though. I can do that.